pallets/staking/src/lib.rs

```diff
@@ -1302,8 +1302,8 @@ decl_event!(
         /// the remainder from the maximum amount of reward.
         /// [era_index, validator_payout, remainder]
         EraPayout(EraIndex, Balance, Balance),
-        /// The staker has been rewarded by this amount. [stash, amount]
-        Reward(AccountId, Balance),
+        /// The staker has been rewarded by this amount. [stash_identity, stash, amount]
+        Reward(IdentityId, AccountId, Balance),
```

Comment:
```suggestion
        Reward(EventDid, AccountId, Balance),
```
plus `.for_event()` at emit.sites.

Reply:
`EventDid` is not currently used in this module. Keeping it as it is for consistency.